test/net/sourceforge/kolmafia/moods/ManaBurnTest.java

```diff
@@ -0,0 +1,53 @@
+package net.sourceforge.kolmafia.moods;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class ManaBurnTest {
+
+  static ManaBurn testMe;
```

Comment:
Evolution.  When I started writing I thought I could use one ManaBurn object for all tests.  Hence static.  When I realized that wasn't going to work I forgot to remove it.  Making it a class variable instead of declaring it in each test method is just lazy - not having to declare testMe when it is initialized.  You do strange things when you are not a touch typist :-)

Reply:
Addressed.